Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2177,10 +2181,12 @@ def gen2018HiMix(fragment,howMuch):
 steps['HARVEST2017'] = merge([ {'--conditions':'auto:run2_data_relval','--era':'Run2_2017','--conditions':'auto:run2_data_promptlike',}, steps['HARVESTD'] ])
 steps['HARVEST2017_L1TEgDQM'] = merge([ {'-s':'HARVESTING:@standardDQM+@ExtraHLT+@miniAODDQM+@L1TEgamma'}, steps['HARVEST2017'] ])
 steps['HARVEST2017_L1TMuDQM'] = merge([ {'-s':'HARVESTING:@standardDQM+@ExtraHLT+@miniAODDQM+@L1TMuon'}, steps['HARVEST2017'] ])
-steps['HARVEST2018'] = merge([ {'--conditions':'auto:run2_data_relval','--era':'Run2_2018','--conditions':'auto:run2_data_promptlike',}, steps['HARVESTD'] ])
+steps['HARVEST2018'] = merge([ {'--era':'Run2_2018','--conditions':'auto:run2_data_promptlike',}, steps['HARVESTD'] ])
```

Comment:
Hi @zhenhu  should HARVEST2018 change conditions?  run2_data_promptlike is GlobalTag for Run2 data 2016H relvals only as mentioned above. While GlobalTag for Run2 data relvals is 'run2_data_relval'  :   '103X_dataRun2_relval_v4'

Reply:
In addition @zhenhu  'run2_data_promptlike' : '103X_dataRun2_PromptLike_v3'  
but acc Luca's email
For data
103X_dataRun2_Prompt_v1
103X_dataRun2_Prompt_v1HEfail_v1

Maybe it can be clarified?

Reply:
Also the MC ones acc Luca is V3 version (see below) but in autoCond.py  is 'phase1_2018_realistic'    : '103X_upgrade2018_realistic_v4'

MC in 10.3 you need:
103X_upgrade2018_realistic_v3HEfail_v1
103X_upgrade2018_realistic_v3

@lpernie  is it expected?

Reply:
Hi @prebello , in Line 2180, the "--conditions" was originally set twice. I don't know why it was like that before. So I removed the first one.

Reply:
Hi @prebello , the GT changes in this PR are expected.  Both the prompt and MC HEfail GTs have been updated by @lpernie for autoCond. The ones used in this PR are the most recent ones. 

NEW QUEUE:
1) 103X_dataRun2_PromptLike_HEfail_Queue
2) 103X_upgrade2018_realistic_HEfail_Queue

Which lead to 2 GTs:
1) 103X_dataRun2_PromptLike_HEfail_v1
(https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/103X_dataRun2_PromptLike_HEfail_v1/103X_dataRun2_PromptLike_v3)
2) 103X_upgrade2018_realistic_HEfail_v1
(https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/103X_upgrade2018_realistic_HEfail_v1/103X_upgrade2018_realistic_v4)

Reply:
so @zhenhu  we cannot consider the ones replied in the thread "HEM-15-16 relval" by Luca, right? there he uses v3 instead of v4

Reply:
Hi @prebello , you are right. The one based on realistic_v3 in the email thread was old. Realistic_v4 will be used for pre3 release, so Luca produced this new HEfail GT based on v4

Reply:
Hi @zhenhu  ok but why did you remove it in line 2180? HARVEST2018 is used for all data relvals that includes L1TEg/MuDQM as well , overall in the code. It has to be understood before you remove it to avoid L1 module crashes in the Harvesting step. Did you check the log files to search for errors related to it?